### PR TITLE
Fixes #134 - update Onboarding document

### LINKED ISSUE #134: UXD: Update Onboarding Guide

### PR DESCRIPTION
**Pull request by monicawheeler**
* Remove blog/team page reference
* Add editors and tools suggestions
* Reorganize questions section
* Add git workflow from wiki
* Reorganize the research and questions section

### CODE REVIEW

File: ux-dev/ux-developer-onboarding.md [hunk 1]
```diff
@@ -10,35 +10,43 @@ and links included within them:
...
+* tmux - Read Brian P. Hogan's book __tmux Productive Mouse-Free Development__  
 
-## Vim / tmux
+IMPORTANT: Regardless of your choice of text editor, it should be easily used by other developers when pairing up and also have the ability to accept `.editorconfig`. See [EditorConfig](http://editorconfig.org/#download) for more information.
 
-Utilize Google first to find out what you don’t know about tmux and Vim.
-* Complete
-  [Vim Adventures](http://vim-adventures.com/)
-* Read __tmux Productive Mouse-Free Development__ - Author, Brian P. Hogan
+*Tools:*
```
Comment (Ctannerweb) [inline ux-dev/ux-developer-onboarding.md]: Lets include the WAVE tool also. https://chrome.google.com/webstore/detail/wave-evaluation-tool/jbbplnpkjmmeebjpijfedlgcdilocofh

File: ux-dev/ux-developer-onboarding.md [hunk 2]
```diff
@@ -10,35 +10,61 @@ and links included within them:
...
-* Any question you can think of!
-* I don’t fully understand this problem even after research, can we pair on it?
+### Editors:
+
+Regardless of your choice of text editor, it should be easily used by other developers when pairing up and also have the ability to accept `.editorconfig`. See [EditorConfig](http://editorconfig.org/#download) for more information.
+
+* Vim -  [Vim Adventures](http://vim-adventures.com/) is a great resource to getting familiar with Vim.
+* tmux - Read Brian P. Hogan's book __tmux Productive Mouse-Free Development__
+* Sublime Text 3
+* VSCode
```
Comment (danrichman) [inline ux-dev/ux-developer-onboarding.md]: There should technically be a space between `VS` and `Code`. 😺
Also, Atom could probably get top billing here, as it's pretty popular.

File: ux-dev/ux-developer-onboarding.md [hunk 3]
```diff
@@ -10,35 +10,61 @@ and links included within them:
...
 * [Beginning a project](https://github.com/dockyard/styleguides/blob/master/ux-dev/beginning-a-project.md)
 
-## Questions to Ask Co-Workers
+## Text Editors and other tools
 
-* Any question you can think of!
-* I don’t fully understand this problem even after research, can we pair on it?
+### Editors:
+
+Regardless of your choice of text editor, it should be easily used by other developers when pairing up and also have the ability to accept `.editorconfig`. See [EditorConfig](http://editorconfig.org/#download) for more information.
```
Comment (danrichman) [inline ux-dev/ux-developer-onboarding.md]: Might be better to say `and must also have the ability to accept...`.

I think lack of EditorConfig support would be considered unacceptable.

File: ux-dev/ux-developer-onboarding.md [hunk 4]
```diff
@@ -10,35 +10,61 @@ and links included within them:
...
+
+* Vim -  [Vim Adventures](http://vim-adventures.com/) is a great resource to getting familiar with Vim.
```
Comment (jsit) [inline ux-dev/ux-developer-onboarding.md]: [Practical Vim](https://pragprog.com/book/dnvim2/practical-vim-second-edition) is a great book to look into if you want to really start to master Vim

File: ux-dev/ux-developer-onboarding.md [hunk 5]
```diff
@@ -10,35 +10,61 @@ and links included within them:
...
+#### Browser/device testing
+
+* Virtual Machines for Windows: to browser test primarily IE11 and Edge. Windows provides [free downloads](https://developer.microsoft.com/en-us/microsoft-edge/tools/vms/) of the test VM's to be installed on Virtual Box.
+* [Genymotion: to test Android devices](https://www.genymotion.com/)
 
-## Vim / tmux
+#### Color a11y tools
+  * [Color Oracle](http://colororacle.org/)
+  * [Contrast Analyser](https://developer.paciellogroup.com/resources/contrastanalyser/)
+  * [WAVE](https://chrome.google.com/webstore/detail/wave-evaluation-tool/jbbplnpkjmmeebjpijfedlgcdilocofh)
```
Comment (jsit) [inline ux-dev/ux-developer-onboarding.md]: [xScope](https://xscopeapp.com) has some good color a11y tools

File: ux-dev/ux-developer-onboarding.md [hunk 6]
```diff
@@ -10,35 +10,61 @@ and links included within them:
...
+  * [ImageAlpha](https://pngmini.com/) (Free)
+  * [SVGOMG](https://jakearchibald.github.io/svgomg/) (Free and Online)
+
+#### Pixel Perfection Tools
+  * [GluePrint](http://glueprintapp.com/) (Free)
+  * [xScope](https://xscopeapp.com/) ($49.99)
+  * [PixelSnap](https://getpixelsnap.com/) ($15)
+
+#### Design Tools
+* [Sketch](https://www.sketchapp.com/) is our design application and how all assets are delivered to development. See your manager for a license!
```
Comment (jsit) [inline ux-dev/ux-developer-onboarding.md]: I think we're also starting to use [Affinity Designer](http://affinity.serif.com/designer) a lot in place of Adobe products?

**Pull request merged**